catalog-rest-service/src/main/resources/ui/src/components/EntityTable/EntityTable.component.tsx

```diff
@@ -93,15 +95,21 @@ const EntityTable = ({
     [searchedColumns]
   );
 
-  const { getTableProps, getTableBodyProps, headerGroups, rows, prepareRow } =
-    useTable(
-      {
-        columns: columns,
-        data,
-        autoResetExpanded: false,
-      },
-      useExpanded
-    );
+  const {
+    getTableProps,
+    getTableBodyProps,
+    headerGroups,
+    rows,
+    prepareRow,
+    toggleAllRowsExpanded,
+  } = useTable(
+    {
+      columns: columns,
```

Comment:
This can be just `columns,`

Reply:
@darth-coder00  columns is only for table header and `data` & `autoResetExpanded` is for table data.